packages/strapi-parts/src/Divider/Divider.stories.mdx

```diff
@@ -12,13 +12,21 @@ import { Box } from '../Box';
 
 # Divider
 
-Dividers are used to separate two pieces of content. The have a generic style in the interface. 
+Dividers are used to separate two pieces of content. They have a generic style in the interface.
 
 **Best practices**
 
 - Depending on what is about to be made, a small spacing left and right can be added.
 - Dividers should not be bolder.
 
+[View source](https://github.com/strapi/parts/tree/develop/packages/strapi-parts/src/Divider)
+
+## Imports
+
+```js
+import { Divider } from '@strapi/parts/Divider';
```

Comment:
Can you also add the ArgsTable ?

Reply:
in the import? 
I think the argsTable is present but there are no props being displayed

Reply:
Ok it might be because it's a styled-component. I'll make an update as soon as this PR is merged.